test/Model.js

```diff
@@ -169,6 +169,20 @@ describe('Model', function (){
     }
     should.not.exist(error);
   });
+  it('Should support schema option updates via constructor', async function () {
+    this.timeout(12000);
+
+    const Wolf3 = dynamoose.model('Wolf32', new dynamoose.Schema({
+      id: Number,
+      name: {
+        type: String,
+      },
+      friends: Map,
+    }), {saveUnknown: ['friends'],});
```

Comment:
this is the key part, the model call has a temp override, and needs to be allowed